pkg/controller/disruption/disruption.go

```diff
@@ -311,10 +311,13 @@ func (dc *DisruptionController) addDb(obj interface{}) {
 }
 
 func (dc *DisruptionController) updateDb(old, cur interface{}) {
-	// TODO(mml) ignore updates where 'old' is equivalent to 'cur'.
-	pdb := cur.(*policy.PodDisruptionBudget)
-	glog.V(4).Infof("update DB %q", pdb.Name)
-	dc.enqueuePdb(pdb)
+	oldPdb := old.(*policy.PodDisruptionBudget)
+	curPdb := cur.(*policy.PodDisruptionBudget)
+	if curPdb.ResourceVersion == oldPdb.ResourceVersion {
```

Comment:
IMO, the `TODO` means pdb's spec instead of version.

Reply:
add an import `apiequality "k8s.io/apimachinery/pkg/api/equality"`
and you can use `apiequality.Semantic.DeepEqual` to compare the two specs here
```go
if !apiequality.Semantic.DeepEqual(curPdb.Spec, oldPdb.Spec) {
```

Reply:
@k82cn @kevin-wangzefeng Thanks for reviewing!